### PR TITLE
re-enable six build on CCI

### PR DESCRIPTION
**Pull request by masci**
### What does this PR do?

Re-enable Six build and test on top of the original CI build&test pipeline

### Notice

There's no change here to any of the `invoke` tasks, meaning that tests are run against the existing embedding strategy.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,24 +25,23 @@ templates:
           # If incremental dep fails, increase the cache gen number
           # in restore_deps AND save_deps
           # See https://github.com/DataDog/datadog-agent/pull/2384
-          - gen12-godeps-{{ .Branch }}-{{ .Revision }}
-          - gen12-godeps-{{ .Branch }}-
-          - gen12-godeps-master-
+          - gen11-godeps-{{ .Branch }}-{{ .Revision }}
+          - gen11-godeps-{{ .Branch }}-
+          - gen11-godeps-master-
```
Comment (olivielpeau) [inline .circleci/config.yml]: shouldn't this be `gen12` instead? (or `gen13` if we need a cache invalidation)

File: .circleci/config.yml [hunk 2]
```diff
@@ -26,24 +25,23 @@ templates:
...
     - save_cache: &save_deps
-        key: gen12-godeps-{{ .Branch }}-{{ .Revision }}
+        key: gen11-godeps-{{ .Branch }}-{{ .Revision }}
```
Comment (olivielpeau) [inline .circleci/config.yml]: ditto

File: .circleci/config.yml [hunk 3]
```diff
@@ -26,24 +25,23 @@ templates:
...
         keys:
           # Cache retrieval is faster than full git checkout
-          - v3-repo-{{ .Revision }}
+          - v2-repo-{{ .Revision }}
     - save_cache: &save_source
-        key: v3-repo-{{ .Revision }}
-    - run: &enter_venv
-        name: add virtualenv to bashrc
-        command: echo "source /go/src/github.com/DataDog/datadog-agent/venv/bin/activate" >> $BASH_ENV
+        key: v2-repo-{{ .Revision }}
```
Comment (olivielpeau) [inline .circleci/config.yml]: ditto, should be `v3` or `v4` I guess

File: .circleci/config.yml [hunk 4]
```diff
@@ -4,19 +4,18 @@ version: 2
 # see https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
 # and https://learnxinyminutes.com/docs/yaml/
 
-experimental:
-  notify:
-    branches:
-      only:
-        - master
```
Comment (olivielpeau) [inline .circleci/config.yml]: any reason to remove this?

File: .circleci/config.yml [hunk 5]
```diff
@@ -4,19 +4,18 @@ version: 2
...
-
 templates:
   job_template: &job_template
     docker:
-      - image: datadog/datadog-agent-runner-circle:latest
+      - image: datadog/datadog-agent-runner-circle:six
```
Comment (olivielpeau) [inline .circleci/config.yml]: note: should be switched back to `latest` at some point, either before or after merging depending on your plan

**Review by olivielpeau — DISMISSED** [review]
A few nits, feel free to merge once answered/addressed!

File: .circleci/config.yml (hunk 5 above)
Comment (masci) [reply .circleci/config.yml]: I'll change `latest` to point to what's `six` now and remove the old Dockerfile in a dedicated PR

**Review by olivielpeau — APPROVED** [review]
(approving again)

**Pull request merged**